src/test/java/org/entando/plugins/pda/controller/ConnectionsControllerIntegrationTest.java

```diff
@@ -0,0 +1,274 @@
+package org.entando.plugins.pda.controller;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.apache.http.entity.ContentType;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.entando.plugins.pda.dto.connection.ConnectionDto;
+import org.entando.plugins.pda.pam.engine.KieEngine;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@ActiveProfiles("test")
+@RunWith(SpringRunner.class)
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class })
+@AutoConfigureWireMock(port = 8089)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class ConnectionsControllerIntegrationTest {
+
+    @Autowired private MockMvc mockMvc;
```

Comment:
Inconsistent placement for annotations. Sometimes it's in a separated line, sometimes it's in the same line as the access modifier.

Reply:
What should be the correct placement?